src/FSharpLint.Core/Text.resx

```diff
@@ -333,4 +333,7 @@
   <data name="RulesFavourConsistentThis" xml:space="preserve">
     <value>Prefer using '{0}' consistently.</value>
   </data>
+  <data name="RulesFavourStaticEmptyFieldsError" xml:space="preserve">
+    <value>Use a static empty field such as String.Empty or List.Empty</value>
```

Comment:
Please be more specific here by using `{0}` inside the string, and injecting "String.Empty" or "List.Empty" depending on what violation happens.

Reply:
(as an example of how this is done, you can look at the rule "FailwithBadUsage")

Reply:
Done